src/components/Skeleton/styles.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../scss/variables';
+
+.skeleton {
+  animation: pulse 1s infinite ease-in-out;
+  border-radius: $border-radius;
+  height: 100%;
+  min-height: 1rem;
```

Comment:
I think this should be `em` because that references the font-size of the current element (`rem` references that of the root). The skeleton container might inherit font-size adjustments and that seems important.

Reply:
Correct, thanks